Activities/Database/ConnectionDialog/ConnectionUIDialog/Workaround/DbWorkarounds.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UiPath.Database.Workaround
+{
+    public static class DbWorkarounds
+    {
+        [DllImport("kernel32.dll", CharSet = CharSet.Auto, SetLastError = true)]
+        private static extern IntPtr LoadLibrary(string libname);
+
+        [DllImport("kernel32.dll", CharSet = CharSet.Auto)]
+        private static extern bool FreeLibrary(IntPtr hModule);
+
+#if NETCOREAPP
+        const string relativePath = @"\..\runtimes\win-x64\native\sni.dll";
```

Comment:
Const should be NamedLikeThis.

Also, let's add the private modifier explicitly, and move it to the top of the class declaration.

Reply:
modified